modules/standard/GMP.chpl

```diff
@@ -51,7 +183,8 @@ module GMP {
      routines if not for ref.
    */
 
-  // initializing rng
+  // Initializing random state 
+  /* */
```

Comment:
In looking at the rest of the generated html, this seems like quite an undertaking.  Would you like some help?